superflore/generators/bitbake/run.py

```diff
@@ -99,10 +99,11 @@ def main():
             )
         else:
             pr_comment = pr_comment or (
-                'Superflore yocto generator began regeneration of package(s)' +
-                ' %s from ROS distro %s from allenh1\'s fork of Meta-ROS ' +
-                'from commit %s.' % (
+                'Superflore yocto generator began regeneration of package(s)'
+                ' %s from ROS distro %s from allenh1\'s fork of Meta-ROS'
```

Comment:
This is missing a parameterization based on #128

Reply:
Honestly, I think the `allenh1` part can be removed.